templates/hooks/certificate_authority/sdk_update_pre_build_request.go.tpl

```diff
@@ -0,0 +1,7 @@
+    latest, err = rm.sdkFind(ctx, latest)
```

Comment:
We don't need to call sdkFind here, the function already receives both desired and latest in the parameters